src/test/scala/sigmastate/CostingSpecification.scala

```diff
@@ -105,6 +107,52 @@ class CostingSpecification extends SigmaTestingData {
     cost("{ val cond = getVar[Boolean](2).get; cond || cond && true || cond }") shouldBe (ContextVarAccess + logicCost * 3 + constCost)
     cost("{ val cond = getVar[Boolean](2).get; cond ^ cond && true ^ cond }") shouldBe (ContextVarAccess + logicCost * 3 + constCost)
     cost("{ val cond = getVar[Boolean](2).get; allOf(Coll(cond, true, cond)) }") shouldBe (ContextVarAccess + logicCost * 2 + constCost)
+    cost("{ val cond = getVar[Boolean](2).get; anyOf(Coll(cond, true, cond)) }") shouldBe (ContextVarAccess + logicCost * 2 + constCost)
+    cost("{ val cond = getVar[Boolean](2).get; xorOf(Coll(cond, true, cond)) }") shouldBe (ContextVarAccess + logicCost * 2 + constCost)
+  }
+
+  property("atLeast costs") {
+    val concrCollCost = proveDlogEvalCost
+    cost("{ atLeast(2, Coll(pkA, pkB, pkB)) }") shouldBe
+      (concrCollCost + proveDlogEvalCost * 3 + logicCost + constCost + constCost)
+  }
+
+  property("allZK costs") {
+    val concrCollCost = proveDlogEvalCost
+    cost("{ pkA && pkB }") shouldBe (concrCollCost + sigmaAndCost * 2)
```

Comment:
@greenhat why ` sigmaAndCost * 2`?

Reply:
@aslesarenko due to `perItemCostOf(...)`, see https://github.com/ScorexFoundation/sigmastate-interpreter/blob/36f8bf749c35ef2424b1e276a7dac1c4fb33c6fd/src/main/scala/sigmastate/eval/RuntimeCosting.scala#L1697
and 
https://github.com/ScorexFoundation/sigmastate-interpreter/blob/36f8bf749c35ef2424b1e276a7dac1c4fb33c6fd/src/main/scala/sigmastate/eval/RuntimeCosting.scala#L218